### ISSUE TITLE
How to kick out a user with a message

### ISSUE THREAD

**Issue by NicholasChan1025** [topfreegames/pitaya#155]
session.go

**Comment by felipejfc** [comment]
you are right, as of now you can't send payloads during Kick, if you want to implemennt this, a PR would be welcome

**Comment by NicholasChan1025** [comment]
I can't find the protobuf file corresponding to kick.pb.go, where can I get that file please

**Comment by chgz** [comment]
in this repo : github.com/topfreegames/pitaya-protos

@NicholasChan1025

**Comment by greatcat-taihe** [comment]
My client code subscribe an "onKicked" event, and server just push "onKicked" event to client before kick it.

**Comment by f2ngwx** [comment]
> My client code subscribe an "onKicked" event, and server just push "onKicked" event to client before kick it.

You may not receive this message
